packages/components/src/components/form/Select/Select.tsx

```diff
@@ -351,50 +344,50 @@ export const Select = ({
-            $elevation={elevation}
-            $isSearchable={isSearchable}
-            $size={size}
-            $minValueWidth={minValueWidth}
-            $isDisabled={isDisabled}
-            $isMenuOpen={isMenuOpen}
-            $isWithLabel={!!label}
-            $isWithPlaceholder={!!placeholder}
-            $hasBottomPadding={hasBottomPadding === true && bottomText === null}
-            {...frameProps}
-        >
-            <ReactSelect
-                ref={selectRef}
-                onKeyDown={onKeyDown}
-                classNamePrefix={reactSelectClassNamePrefix}
-                openMenuOnFocus
-                closeMenuOnScroll={closeMenuOnScroll}
-                menuPosition="fixed" // Required for closeMenuOnScroll to work properly when near page bottom
-                menuPortalTarget={menuPortalTarget}
-                styles={createSelectStyle(theme, elevation, isRenderedInModal)}
-                onChange={handleOnChange}
-                isSearchable={isSearchable}
-                menuIsOpen={isMenuOpen}
-                isDisabled={isDisabled}
-                menuPlacement="auto"
-                placeholder={placeholder || ''}
-                {...props}
-                components={memoizedComponents}
-            />
-
-            {label && (
-                <SelectLabel $size={size} $isDisabled={isDisabled}>
-                    {label}
-                </SelectLabel>
-            )}
-
-            {bottomText && (
-                <BottomText inputState={inputState} isDisabled={isDisabled}>
-                    {bottomText}
-                </BottomText>
-            )}
-        </Wrapper>
+        <FormCell {...formCellProps} isDisabled={isDisabled}>
+            <SelectWrapper
+                $isClean={isClean}
+                $elevation={elevation}
+                $isSearchable={isSearchable}
+                $size={size}
+                $minValueWidth={minValueWidth}
+                $isDisabled={isDisabled}
+                $isLoading={isLoading}
+                $isMenuOpen={isMenuOpen}
+                $isWithLabel={!!label}
+                $isWithPlaceholder={!!placeholder}
+            >
+                <ReactSelect
+                    ref={selectRef}
+                    onKeyDown={onKeyDown}
+                    classNamePrefix={reactSelectClassNamePrefix}
+                    openMenuOnFocus
+                    closeMenuOnScroll={closeMenuOnScroll}
+                    menuPosition="fixed" // Required for closeMenuOnScroll to work properly when near page bottom
+                    menuPortalTarget={menuPortalTarget}
+                    styles={createSelectStyle(theme, elevation, isRenderedInModal)}
+                    onChange={handleOnChange}
+                    isSearchable={isSearchable}
+                    menuIsOpen={isMenuOpen}
+                    isDisabled={isDisabled}
+                    menuPlacement="auto"
+                    placeholder={placeholder || ''}
```

Comment:
nit `??` would make more sense, but it's only for semantics, not function :shrug: